src/integrationTest/java/com/mongodb/spark/sql/connector/MongoCatalogTest.java

```diff
@@ -306,7 +308,8 @@ void loadTableTest() throws NoSuchTableException {
         new MongoTable(
             new MongoNamespace(TEST_DATABASE_NAME, TEST_COLLECTION_NAME),
             null,
-            getConnectionProviderOptions()),
+            new MongoConnectionProvider(
```

Comment:
The `MongoConnectionProvider` now takes the `MongoConfig` rather than the raw options.